backend/entities/serializers.py

```diff
@@ -1,18 +1,189 @@
 from rest_framework import serializers
-
 from .models import *
+from content.models import Resource, Task, Topic
+from events.models import Event
+from authentication.models import User
+from utils.utils import (
+    validate_creation_and_deletion_dates,
+    validate_flags_number,
+    validate_empty,
+    validate_object_existence,
+)
 
 
 class OrganizationSerializer(serializers.ModelSerializer):
     class Meta:
         model = Organization
-        fields = (
-            "id",
-            "name",
-            "tagline",
-            "social_accounts",
-            "total_flags",
-            "created_by",
-            "creation_date",
-            "deletion_date",
+        fields = "__all__"
+
+    def validate(self, data):
+        validate_empty(data["name"], "name")
+        validate_empty(data["tagline"], "tagline")
+        validate_empty(data["social_accounts"], "social_accounts")
+        validate_flags_number(data)
+        validate_creation_and_deletion_dates(data)
+
+        return data
+
+
+class OrganizationApplicationStatusSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationApplicationStatus
+        fields = "__all__"
+
+
+class OrganizationApplicationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationApplication
+        fields = "__all__"
+
+    def validate(self, data):
+        validate_empty(data["status"], "status")
+        validate_creation_and_deletion_dates(data)
+        return data
+
+
+class OrganizationEventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationEvent
+        fields = "__all__"
+
+    def validate(self, data):
+        if data["org_id"] == "" or data["event_id"] == "":
+            raise serializers.ValidationError(
+                "org_id and event_id cannot be empty, they must be filled so that the event can be added to the organization"
+            )
+        
+        validate_object_existence(Organization, data["org_id"], "org_id does not exist")
+        validate_object_existence(Event, data["event_id"], "event_id does not exist")
+
+        return data
+
+
+class OrganizationMemberSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationMember
+        fields = "__all__"
+
+    def validate(self, data):
```

Comment:
This one here is also not necessary. Would be great to get feedback on the various roles we'll include for orgs, but generally we just have these three right now as special roles with more permissions :)